### PR TITLE
Enable UNS

### PR DESCRIPTION
**Pull request by sudoryan**
Rename CNS to UNS class
Switch CNS readings to the new ProxyReader
Migrate to the new network config https://github.com/unstoppabledomains/uns/blob/main/uns-config.json
Updates CNS tests to test against UNS rinkeby network

### CODE REVIEW

File: src/test/java/com/unstoppabledomains/resolution/ResolutionTest.java [hunk 1]
```diff
@@ -311,25 +312,27 @@ public void buildingDefaultProvider() throws Exception {
...
         String omni = resolution.getMultiChainAddress(domainWithMultiChainRecords, "usdt", "omni");
-        assertEquals("19o6LvAdCPkjLi83VsjrCsmvQZUirT4KXJ", omni);
+        assertEquals("1KvzMF2Vjy14d6JGY7dG7vjT5kfpmzSQXM", omni);
+        String eos = resolution.getMultiChainAddress(domainWithMultiChainRecords, "usdt", "eos");
+        assertEquals("karaarishmen", eos);
 
         
     }
 
-    @Test
```
Comment (hivtushok) [inline src/test/java/com/unstoppabledomains/resolution/ResolutionTest.java:326]: Can we test this or remove it completely?

**Review by hivtushok — APPROVED** [review]

File: src/test/java/com/unstoppabledomains/resolution/ResolutionTest.java (hunk 1 above)
Comment (hivtushok) [reply src/test/java/com/unstoppabledomains/resolution/ResolutionTest.java:326]: Everything else is good to me 👍🏼

File: src/test/java/com/unstoppabledomains/TestUtils.java [hunk 2]
```diff
@@ -10,8 +10,8 @@
 
 public class TestUtils {
 
-    public static final String TESTING_CNS_PROVIDER_URL = "https://mainnet.infura.io/v3/e0c0cb9d12c440a29379df066de587e6";
-    public static final String TESTING_INFURA_CNS_PROJECT_ID = "e0c0cb9d12c440a29379df066de587e6";
+    public static final String TESTING_UNS_PROVIDER_URL = "https://rinkeby.infura.io/v3/e0c0cb9d12c440a29379df066de587e6";
```
Comment (DeRain) [inline src/test/java/com/unstoppabledomains/TestUtils.java:13]: Good idea with migration of all tests to testnet. 

Could you add more domains with new TLD's into tests?

**Review by DeRain — CHANGES_REQUESTED** [review]
Looks good

File: src/test/java/com/unstoppabledomains/resolution/ResolutionTest.java (hunk 1 above)
Comment (DancingAxolotl) [reply src/test/java/com/unstoppabledomains/resolution/ResolutionTest.java:326]: @sudoryan TokenURIs are different in testnet, but they should be working now. See https://github.com/unstoppabledomains/resolution/blob/3e43971133285c2832fba6004e12349b65cb4ba1/src/tests/Uns.test.ts#L692

**Review by DancingAxolotl — APPROVED** [review]

**Pull request merged**